src/main/scala/lc/Main.scala

```diff
@@ -40,32 +41,35 @@ class Captcha(throttle: Int) {
   }
 
   def getCaptcha(id: Id): Array[Byte] = {
+    var image :Array[Byte] = null
+    var blob: Blob = null
```

Comment:
Use `if..else` statements as expressions, so that you won't need `var`.

Reply:
This doesn't work as expected. Keeping it as it is for now.